torch/csrc/autograd/engine.cpp

```diff
@@ -256,6 +257,11 @@ void Engine::decrement_non_reentrant_thread_count() {
 }
 
 void Engine::thread_init(int device, const std::shared_ptr<ReadyQueue>& ready_queue, bool should_increment) {
+// On Windows, std::set_terminate is used on a per-thread basis. So we need to
```

Comment:
Do we also have a thread pool in c10 where this should be added?

Reply:
I'd be glad if you could tell where they are used.